doc/en/conf.py

```diff
@@ -170,13 +171,8 @@
 linkcheck_workers = 5
 
 
-_repo = "https://github.com/pytest-dev/pytest"
 extlinks = {
     "bpo": ("https://bugs.python.org/issue%s", "bpo-%s"),
```

Comment:
Since BPO is shut down, we might want to switch it to point to cpython github now (not necessary for this PR though).

Reply:
Fair, I wasn't sure. But yeah, could be separate.

Reply:
Done in #12533.